src/strategies/clqdr-balance-with-lp/index.ts

```diff
@@ -0,0 +1,139 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller, call } from '../../utils';
+
+export const author = 'liquiddriver';
```

Comment:
Should be your github user name or the org username
```suggestion
export const author = 'LiquidDriver-finance';
```

Reply:
Done